apps/modernization-ui/src/authorization/ProtectedLayout.tsx

```diff
@@ -14,7 +14,7 @@ const ProtectedLayout = () => {
     const navigate = useNavigate();
 
     const handleIdle = () => {
-        navigate('/expired');
+        navigate('/logout');
```

Comment:
issue:  If a user idles passed the warning time they should be navigated to the `/nbs/logout` path.  Since this is an external route (not within the react router) the navigation has to done by setting `window.location.href` to `/nbs/logout`

Reply:
fixed